### PR TITLE
docs: update readme Korea version

### PR DESCRIPTION
**Pull request by Hoseong-Seo**
### Issue # (if applicable)

Closes #<issue number here>.

### Description of changes

1. Changed the READMEs  Korea version as refreshed the READMEs to reflect some important updates.
2. RoadMap link update.
### Checklist

- [ ] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/awslabs/sbt-aws/blob/main/CONTRIBUTING.md)
---

*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the project license.*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ You may also find help on these community resources:
 
 ## Roadmap
 
-The [SBT for AWS Roadmap project board](PLACEHOLDER) lets developers know about our upcoming features and priorities to help them plan how to best leverage the CDK and identify opportunities to contribute to the project. 
+The [SBT for AWS Roadmap project board](https://github.com/orgs/awslabs/projects/167) lets developers know about our upcoming features and priorities to help them plan how to best leverage the CDK and identify opportunities to contribute to the project.
```
Comment (suhussai) [inline README.md:57]: we recently discovered that this is not public.

**Review by suhussai — APPROVED** [review]
Minor comment regarding roadmap link.

File: README.md (hunk 1 above)
Comment (Hoseong-Seo) [reply README.md:57]: This change is only for internal for now. Update it and wait for being public, if a new URL will be needed, we'll update it.
And Korean ReadMe applied the latest version.

**Comment by Hoseong-Seo** [comment]
Closing this PR as it's outdated. I'll open a new PR with latest update soon.

**Pull request closed without merging**